app/web_hub.go

```diff
@@ -523,7 +528,7 @@ func (h *Hub) Start() {
 	doRecover = func() {
 		if !h.ExplicitStop {
 			if r := recover(); r != nil {
-				mlog.Error(fmt.Sprintf("Recovering from Hub panic. Panic was: %v", r))
+				mlog.Error("Recovering from Hub panic.", mlog.String("panic", r.(string)))
```

Comment:
I think we don't know the type here so we should use `mlog.Any`

Reply:
done